cmd/traffic/cmd/manager/internal/cluster/info.go

```diff
@@ -77,14 +87,20 @@ func NewInfo(ctx context.Context) Info {
 	}
 
 	client := ki.CoreV1()
-	// Get the clusterID from the default namespaces
-	// We use a default clusterID because we don't want to fail if
-	// the traffic-manager doesn't have the ability to get the namespace
-	ns, err := client.Namespaces().Get(ctx, "default", metav1.GetOptions{})
+	// Get the clusterID from the default namespace, or from the manager's namespace if
+	// the traffic-manager doesn't have access to the default namespace.
+	nsForID := "default"
+	ns, err := client.Namespaces().Get(ctx, nsForID, metav1.GetOptions{})
+	if err != nil {
+		nsForID = env.ManagerNamespace
```

Comment:
could we dlog.warn or info on the err? i can see it being useful if the user expects to be able to get the default ns